src/fullyautomatednutcracker/cogs/antiselfdeprecation.py

```diff
@@ -1,6 +1,9 @@
 from discord.ext import commands
 import asyncio
 
+import time
```

Comment:
normally I would gripe about import order and the amount of new lines, but I'll let it slide.